src/blueapi/core/context.py

```diff
@@ -61,7 +60,7 @@ def find_device(self, addr: str | list[str]) -> Device | None:
         Find a device in this context, allows for recursive search.
 
         Args:
-            addr (Union[str, List[str]]): Address of the device, examples:
+            addr (str | list[str]): Address of the device, examples:
```

Comment:
Nit: I don't tend to bother with type annotations at all in docstrings, since I already put them in the function signatures and AFAIK that's enough for tools like autodoc.